components/table-row/table-row-notifications.js

```diff
@@ -36,7 +36,7 @@ function TableRowNotifications({certification, comment, warning}) {
 
 TableRowNotifications.propTypes = {
   certification: PropTypes.string,
-  comment: PropTypes.string,
+  comment: PropTypes.node,
```

Comment:
On doit avoir un avertissement dans le cas des numéros non ?
On pourrait utiliser `oneOfType` :
```suggestion
  comment: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```